.travis.yml

```diff
@@ -55,3 +58,5 @@ script:
   - if [[ $XMLLINT == "1" ]]; then diff -B ./src/Standards/PSR12/ruleset.xml <(xmllint --format "./src/Standards/PSR12/ruleset.xml"); fi
   - if [[ $XMLLINT == "1" ]]; then diff -B ./src/Standards/Squiz/ruleset.xml <(xmllint --format "./src/Standards/Squiz/ruleset.xml"); fi
   - if [[ $XMLLINT == "1" ]]; then diff -B ./src/Standards/Zend/ruleset.xml <(xmllint --format "./src/Standards/Zend/ruleset.xml"); fi
+  # Run PHPStan
+  - if [[ $PHPSTAN == "1" ]]; then composer require --dev phpstan/phpstan:0.10.6 && php vendor/phpstan/phpstan/bin/phpstan analyse src --level=0 --configuration=phpstan/config.neon --autoload-file=phpstan/autoload.php; fi
```

Comment:
Is there a reason to hard-code the version number here? Is this something I should remove in the future once a specific version is released?

Reply:
It depends on how much you want to lock version... I used to fix version even with patch version.

So `^0.10` is fine for you then?

Reply:
@gsherwood